### PR TITLE
Choreo Connect - ELK Analytics Configuration

### PR DESCRIPTION
**Pull request by Amila-Rukshan**
## Purpose

Added configuration information for CC - ELK Analytics.

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md [hunk 1]
```diff
@@ -78,6 +78,119 @@ Here are some of the graphs generated in Choreo cloud.
 
 [![Choreo Analytics Latency]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)
 
+## ELK Analytics for Choreo Connect
```
Comment (suksw) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md:81]: Shall we introduce the two main ways at the top of the page? Right after the intro paragraph? We can link the two subsections too. Then the user wouldn't have to scroll to find it.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md [hunk 2]
```diff
@@ -3,8 +3,12 @@
 Choreo Connect Analytics provides reports, dashboards, statistics, and graphs for the APIs deployed on Choreo Connect.
 WSO2 Choreo Connect has the capability to publish events to the Choreo platform in order to generate analytics. This page describes the feature and explains how the feature could be used to generate useful analytics in order to gain important insights into the APIs deployed on the Choreo Connect. To learn more concepts on analytics, follow the [concepts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics/).
 
+There are two ways you can configure Analytics,
 
-## Configuring Analytics for Choreo Connect
+1. [Choreo Portal Analytics for Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/#choreo-portal-analytics-for-choreo-connect)
+2. [ELK Analytics for Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/#elk-analytics-for-choreo-connect)
```
Comment (suksw) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md:9]: Links within the same page can be shortened like this https://github.com/wso2/docs-apim/blob/63829dc504aff1ff82e7a360c0dae0ba53d2e23a/en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md?plain=1#L13

**Review by pubudu538 — APPROVED** [review]

**Comment by Mariangela** [comment]
The changes introduced in the PR review will be sent via a separate PR therefore, I am closing this PR.

**Comment by Mariangela** [comment]
Reviewed and updated this PR via PR #5845 [1]

[1] https://github.com/wso2/docs-apim/pull/5845

**Pull request closed without merging**